### ISSUE TITLE
Page region missing an accessible name - SiteImprove

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9041]
Any page section with the role of region should also have an accessible name to identify it.

SiteImprove flags as ARIA

This is an issue on the Department Codes page as well as the Correspondence which are both developed by us.

Department Codes
https://my2.siteimprove.com/Inspector/995867/A11Y/Page?pageId=25148792236&impmd=264ED37B65FA6186129334A508391970&issue=cantTell+failed&wcag=twopointone#/sia-r40/failed/OXU2MFU1cnQ=;OWI2OTljMGY=

Correspondence
https://my2.siteimprove.com/Inspector/995867/A11Y/Page?pageId=84066828553&impmd=264ED37B65FA6186129334A508391970&issue=cantTell+failed&wcag=twopointone#/sia-r40/failed/MVhnK2NjOWk=;ODc2ZDQxMmQ=

**Comment by briand44** [comment]
Looks like there are some instances in viewbooks. Not sure if ^^ would catch those or not. 

https://my2.siteimprove.com/Inspector/1230859/A11y/Page?pageId=83112006280&impmd=264ED37B65FA6186129334A508391970&conf=aria#/sia-r40/failed/QWlERUc3amw=;ZjNiZTI5ZmY=

**Comment by joewhitsitt** [comment]
>maybe it would make sense to find instances where we are adding role="region" but don't have a label and fix all of them.

agreed. the "after content" one, i thought it made sense to convert to `aside` but it looks like work was only focused to the prefooter. https://github.com/uiowa/uiowa/issues/6297

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complicated - may be different solutions per instance

 **Discussion:**
 - Address all instances across SiteNow, not just Registrar
 - keep viewbook separate slice since in different repos, possibly bundle other viewbook accessibility issues at that time.
 - solutions may be unique per instance
   - maybe some should not have `role="region"`
   - add aria labels where appropriate
 - use https://github.com/uiowa/uiowa/issues/9041#issuecomment-3513621190 to identify instances
 - May make sense to have a future slice to audit usage of `Role` - https://github.com/uiowa/uiowa/issues/1119

**Proposed solution or definition of done:** 
- Identify instances of `Role="region"`
- Add aria labels if using proper role
- adjust/remove role if improper use
- scoped to SiteNow (not viewbooks)
